lib/plugins/inspec-parallel/lib/inspec-parallel/cli.rb

```diff
@@ -17,6 +17,8 @@ class CLI < Inspec.plugin(2, :cli_command)
       desc: "Prints all thor options on dry run"
     option :jobs, aliases: :j, type: :numeric,
       desc: "Number of jobs to run parallely"
+    option :daemon, aliases: :d, type: :boolean,
```

Comment:
Yeah that's true. I had the same opinion about it, but I also felt that option name as "bg" should be not creating confusion too because we are using it as a shortcut for "background". Alternatively, having a 2 character alias using thor seemed not possible.
But since you feel going with "bg" is clear enough with the option name, I will go for this change. Thanks!